integration_tests/pages/apply/about_the_person/equality_and_diversity/asianBackgroundPage.ts

```diff
@@ -0,0 +1,20 @@
+import { Cas2Application as Application } from '../../../../../server/@types/shared/models/Cas2Application'
```

Comment:
Slightly annoying, but we should update these types to use `Cas2V2Application` instead.

Reply:
ah yeah, good spot!